charset_normalizer/api.py

```diff
@@ -67,9 +63,9 @@ def from_bytes(
         )
 
     if not explain:
-        logger.setLevel(logging.CRITICAL)
+        charset_normalizer.set_logging_handler(level=logging.CRITICAL)
```

Comment:
This would immediately add the `StreamHandler`, isn't that counterproductive? We must find a way to call fn `set_logging_handler()` only when `explain=True` and when `explain=False` we must verify that we used the function already, either to restore to the initial logger state or simply to change the verbose level.